internal/provider/provider_test.go

```diff
@@ -30,6 +30,10 @@ func testAccPreCheck(t *testing.T) {
 	requireEnvironmentVariables(t, RedisCloudUrlEnvVar, rediscloud_api.AccessKeyEnvVar, rediscloud_api.SecretKeyEnvVar)
 }
 
+func testAccAwsPreExistingCloudAccountPreCheck(t *testing.T) {
```

Comment:
This function should be used in all of the tests that use `AWS_TEST_CLOUD_ACCOUNT_NAME` such as `TestAccDataSourceRedisCloudSubscription_basic`

Reply:
Good catch, I was meant to do that but forgot